src/init.cpp

```diff
@@ -359,14 +359,17 @@ void SetupServerArgs()
         "-dbcrashratio", "-forcecompactdb",
         // GUI args. These will be overwritten by SetupUIArgs for the GUI
         "-choosedatadir", "-lang=<lang>", "-min", "-resetguisettings", "-splash", "-uiplatform"};
-
+	// VBK
+	gArgs.AddArg("-bfiendpoint", "Default end point for BFI setup.", ArgsManager::ALLOW_STRING, OptionsCategory::OPTIONS);
```

Comment:
could you pls setup your editor to use spaces instead of tabs?

Reply:
Changed for future reference.